### DISCUSSION TITLE
Jupyter doesn't start up - missing home?

### DISCUSSION THREAD

**Discussion by martin-koehler** [ARM-Development/comble-mip#41] [General]
I have tried a few times to start Jupyter by clicking the "rocket" sign on top of the webpage "Check my SCM run". The error message say that this files is not available:
/export/data/home/martinkoehler
/export/home/home-users/martinkoehler
What do I have to do to fix this? Thanks!

**Comment by twjuliano** [comment]
Hi @martin-koehler,
I presume the issue is related to you not having access to the JupyterHub. Have you applied yet? You can follow our instructions on [this page](https://arm-development.github.io/comble-mip/notebooks/howto/workbench_instructions.html) to apply for access.
Please let me know.
Thanks,
Tim
